src/actions/roles.js

```diff
@@ -65,12 +65,12 @@ export function loadRolesIfNeeded(roles) {
             pendingRoles.add(role);
         }
         if (!state.entities.general.serverVersion) {
-            setPendingRoles(pendingRoles);
+            setPendingRoles(pendingRoles)(dispatch, getState);
```

Comment:
I know we already merged this but as a reference for the future this can be done like

```js
dispatch(setPendingRoles(pendingRoles));
```

Reply:
thanks